heron/schedulers/src/java/org/apache/heron/scheduler/aurora/AuroraField.java

```diff
@@ -35,5 +35,8 @@
   TIER,
   TOPOLOGY_ARGUMENTS,
   TOPOLOGY_NAME,
-  TOPOLOGY_PACKAGE_URI
+  TOPOLOGY_PACKAGE_URI,
+  AURORA_METADATA_1,
```

Comment:
1. Why three such aurora configs, not less or more? 

2. Is it possible to assign them meaningful names instead of 1,2,3?

Reply:
Agreed.

Also, is it possible to have a more general metadata argument like "SCHEDULE_META" so that this is for all schedulers? It is not ideal to have AURORA specific configs in heron/spi/common/Key.java to me.

Reply:
_just speculating:_ the `SCHEDULER_PROPERTIES     ("heron.scheduler.properties", Type.PROPERTIES),` line in `Key.java` looks interesting:
 * does `Type.PROPERTIES` allow you do take a set/map of things instead of a string?
 * can `SCHEDULER_PROPERTIES` be used instead of adding these new options?

Reply:
Let us use one parameter and comma separated. heron.scheduler.properties is a good one.

Reply:
updated according to @kramasamy suggestion